### PR TITLE
Added "considerations"

### PR DESCRIPTION
**Pull request by azarboon**
*Issue #, if available:*

*Description of changes:*

Added considerations so readers can take a more informed decision.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,6 +169,12 @@ Please note that `sam local` starts a Lambda Runtime Interface Emulator on port
 
 The Lambda Web Adapter also supports all non-HTTP event triggers, such as SQS, SNS, S3, DynamoDB, Kinesis, Kafka, EventBridge, and Bedrock Agents. The adapter forwards the event payload to the web application via http post to a path defined by the `AWS_LWA_PASS_THROUGH_PATH` environment variable. By default, this path is set to `/events`. Upon receiving the event payload from the request body, the web application should processes it and returns the results as a JSON response. Please checkout [SQS Express.js](examples/sqs-expressjs) and [Bedrock Agent FastAPI in Zip](examples/bedrock-agent-fastapi-zip) examples.
 
+## Considerations
+
+- Implementing this approach may require refactoring to accommodate integration discrepancies.
```
Comment (mxfactorial) [inline README.md:174]: already documented in the readme usage section and repo examples

File: README.md [hunk 2]
```diff
@@ -169,6 +169,12 @@ Please note that `sam local` starts a Lambda Runtime Interface Emulator on port
...
+- Coarser-grained alerting and access controls compared to function-level segmentation.
```
Comment (mxfactorial) [inline README.md:175]: the adapter doesn't impose these limits

File: README.md [hunk 3]
```diff
@@ -169,6 +169,12 @@ Please note that `sam local` starts a Lambda Runtime Interface Emulator on port
...
+- Potential cold start penalties, especially with large web framework dependencies.
```
Comment (mxfactorial) [inline README.md:176]: this refers to lambda and not the adapter

**Review by mxfactorial — COMMENTED** [review]
please close

**Comment by bnusunny** [comment]
Closed as this is a architecture decision, not about this tool.

**Pull request closed without merging**